src/main/java/dev/sefiraat/cultivation/implementation/utils/Utils.java

```diff
@@ -0,0 +1,14 @@
+package dev.sefiraat.cultivation.implementation.utils;
+
+import org.bukkit.Bukkit;
+
+public class Utils {
```

Comment:
While arguably fine, I’m not the biggest fan of the package and the class being named the same when there are other classes in said package - could this be renamed to `GeneralUtils`?

Reply:
Sure! I wasn't sure how to do the utils and assumed this would need changing, just had no idea where to put the method lol

Reply:
I agree, I don’t really think GeneralUtils is much better either ha, but it works and it needed so it’s all good :)